<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureWidget</class>
 <widget class="QWidget" name="PictureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAddImages">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemoveImage">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMoveUp">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/up.png</normaloff>:/icons/icons/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonMoveDown">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/down.png</normaloff>:/icons/icons/down.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Picture Preview:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonGoLive">
       <property name="text">
        <string>Go Live (F5)</string>
       </property>
       <property name="icon">
        <iconset resource="softprojector.qrc">
         <normaloff>:/icons/icons/go_live.png</normaloff>:/icons/icons/go_live.png</iconset>
       </property>
       <property name="shortcut">
        <string>F5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelPreview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Picture Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPixInfo">
       <property name="text">
        <string>Picture Information</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="softprojector.qrc"/>
 </resources>
 <connections/>
</ui>
